### PR TITLE
run Flow in CI

### PR DESCRIPTION
**Pull request by jpdriver**
⚠️ I'm expecting this to break our builds

--

Flow is erroring but not failing this build :(
https://travis-ci.org/FormidableLabs/eslint-plugin-react-native-a11y/jobs/559297165

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		"build": "rimraf lib && babel src --out-dir lib --copy-files",
 		"coveralls": "cat ./reports/lcov.info | coveralls",
 		"create": "node ./scripts/create-rule",
-		"flow": "if [ ! -e ./.flowconfig ]; then echo \"Could not find .flowconfig\"; else flow; test $? -eq 0 -o $? -eq 2; fi",
+		"flow": "flow",
```
Comment (kelset) [inline package.json]: shouldn't this be `flow check`? 🤔

Comment (jpdriver) [reply package.json]: `flow check` can only be used after `flow start` to spawn the server. this is a one-hit command that does both 😇

**Comment by jpdriver** [comment]
This is now erroring because in the `accessibility-label` rule we check `node.parent` for an `accessibilityLabel` prop.

`node` there is typed as a `JSXOpeningElement`; but `node.parent` is undefined in the `JSXOpeningElement` type -- and hence we get the Flow error.

Although this is (no longer?) mentioned in the React Native docs, I checked this scenario using RN0.60.3 and the iOS Accessibility Inspector -- and it is indeed valid.

@alex-saunders previously bumped into the type error [here](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/pull/26#discussion_r202036516) and it seems like it's just gone un-noticed since.

to get around this, perhaps we could do

```
type Node = JSXOpeningElement & {
  parent: Object
}
```

I know that might not be ideal -- but I'm not sure about the correct type for `node.parent` and I don't want to type it manually property by property.

Taking the hit on a loose Object type here would solve our Flow error and keep the value of using both Flow and `JSXOpeningElement` elsewhere; although admittedly it does feel kind of wrong to be using `JSXOpeningElement` at all here if it doesn't _actually_ reflect the object shape without extensions..

File: package.json (hunk 1 above)
Comment (kelset) [reply package.json]: TIL

**Comment by alex-saunders** [comment]
@jpdriver extending the `JSXOpeningElement` type feels a bit iffy for me. As far as I can see, this is only erroring in 1 place? If so, could we introduce another `$FlowFixMe` to stop it complaining for now?
Could be worth opening an issue on ast-types-flow to introduce the `parent` field to `JSXOpeningElement`, although that project looks fairly stagnant..

**Comment by jpdriver** [comment]
ok will do that instead

**Pull request merged**